### DISCUSSION TITLE
rna and protein docking failed!!!!

### DISCUSSION THREAD

**Discussion by mmjwxbc** [RosettaCommons/rosetta#180] [Rosetta]
when I user the Rosetta/demos/public/rnp_structure_prediction, I run the command -- rna_denovo, it was successful. But when i change the rna seq and the rna secstructure in the rna & protein complex, it reported an error;

There are my seq, structure and flag above
[error.log](https://github.com/user-attachments/files/17183431/error.log)
[flags.txt](https://github.com/user-attachments/files/17183440/flags.txt)
[seq.fasta.txt](https://github.com/user-attachments/files/17183441/seq.fasta.txt)
[seq.txt](https://github.com/user-attachments/files/17183442/seq.txt)
[unbound_protein.pdb.txt](https://github.com/user-attachments/files/17183443/unbound_protein.pdb.txt)

**Comment by roccomoretti** [comment]
It looks like the error you're getting is `Could not match residue number  124 and chain "A" and segid "    "`

I'm guessing that the issue you're running into is a "PDB numbering" versus "pose numbering" issue. I notice on the seq.fasta file, you have the annotation `A:3-125 B:126-242 C:245-269`. While this matches the nominal residue numbering in the PDB file, it doesn't match the sequential (pose) numbering. That is, chain A does not start with the 3 amino acid in the fasta file, nor does it end with the 125th. Instead, it starts with the 1st and ends with the 123rd. This means that the 124th residue (when counting sequentially from 1) is actually in chain B and not in chain A.

Try adjusting your seq.fasta annotation such that it counts sequentially from 1. You may want to also try renumbering your PDB such that it starts at 1 and has sequentially numbered residues. (An easy way to do that is to run `score_jd2.linuxgccrelease -s input.pdb -out:pdb -renumber_pdb` ... the resulting input_0001.pdb will be renumbered according to Rosetta's pose numbering.)
